CHANGELOG.md

```diff
@@ -1 +1 @@
-See [CHANGELOG/README.md](./CHANGELOG/README.md).
\ No newline at end of file
+See the [1.17 changelog in the master branch](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.17.md).
```

Comment:
Change 1.17 -> 1.18.

It doesn't matter if the 1.18 changelog doesn't exist yet upstream, we'll create it soon enough.